src/main/java/edu/wpi/first/nativeutils/WPINativeUtils.java

```diff
@@ -13,5 +15,21 @@ public void apply(Project project) {
     nativeExt.addWpiExtension();
 
     project.getPluginManager().apply(RpathRules.class);
-  }
+
+    if (project.hasProperty("developerID")) {

```

Comment:
The developerId property was just for the allwpilib build; the plugin should have a way to enable signing and define the developer key arbitrarily through the buildscript.

Also would be good to only enable this feature on macOS.